zhaquirks/sinope/__init__.py

```diff
@@ -29,38 +32,90 @@
     (SHORT_PRESS, TURN_ON): {
         ENDPOINT_ID: 1,
         CLUSTER_ID: 65281,
-        COMMAND: COMMAND_BUTTON_SINGLE,
-        ARGS: {ATTRIBUTE_ID: 84, ATTRIBUTE_NAME: ATTRIBUTE_ACTION, VALUE: 2},
+        COMMAND: COMMAND_M_INITIAL_PRESS,
+        ARGS: {
+            ATTRIBUTE_ID: 84,
+            ATTRIBUTE_NAME: ATTRIBUTE_ACTION,
+            BUTTON: TURN_ON,
+            VALUE: 1,
+        },
     },
     (SHORT_PRESS, TURN_OFF): {
         ENDPOINT_ID: 1,
         CLUSTER_ID: 65281,
-        COMMAND: COMMAND_BUTTON_SINGLE,
-        ARGS: {ATTRIBUTE_ID: 84, ATTRIBUTE_NAME: ATTRIBUTE_ACTION, VALUE: 18},
+        COMMAND: COMMAND_M_INITIAL_PRESS,
+        ARGS: {
+            ATTRIBUTE_ID: 84,
+            ATTRIBUTE_NAME: ATTRIBUTE_ACTION,
+            BUTTON: TURN_OFF,
+            VALUE: 17,
+        },
+    },
+    (SHORT_RELEASE, TURN_ON): {
+        ENDPOINT_ID: 1,
+        CLUSTER_ID: 65281,
+        COMMAND: COMMAND_M_SHORT_RELEASE,
+        ARGS: {
+            ATTRIBUTE_ID: 84,
+            ATTRIBUTE_NAME: ATTRIBUTE_ACTION,
+            BUTTON: TURN_ON,
+            VALUE: 2,
+        },
+    },
+    (SHORT_RELEASE, TURN_OFF): {
+        ENDPOINT_ID: 1,
+        CLUSTER_ID: 65281,
+        COMMAND: COMMAND_M_SHORT_RELEASE,
+        ARGS: {
+            ATTRIBUTE_ID: 84,
+            ATTRIBUTE_NAME: ATTRIBUTE_ACTION,
+            BUTTON: TURN_OFF,
+            VALUE: 18,
+        },
     },
     (DOUBLE_PRESS, TURN_ON): {
         ENDPOINT_ID: 1,
         CLUSTER_ID: 65281,
-        COMMAND: COMMAND_BUTTON_DOUBLE,
-        ARGS: {ATTRIBUTE_ID: 84, ATTRIBUTE_NAME: ATTRIBUTE_ACTION, VALUE: 4},
+        COMMAND: COMMAND_M_MULTI_PRESS_COMPLETE,
+        ARGS: {
+            ATTRIBUTE_ID: 84,
+            ATTRIBUTE_NAME: ATTRIBUTE_ACTION,
+            BUTTON: TURN_ON,
+            VALUE: 4,
+        },
     },
     (DOUBLE_PRESS, TURN_OFF): {
         ENDPOINT_ID: 1,
         CLUSTER_ID: 65281,
-        COMMAND: COMMAND_BUTTON_DOUBLE,
-        ARGS: {ATTRIBUTE_ID: 84, ATTRIBUTE_NAME: ATTRIBUTE_ACTION, VALUE: 20},
+        COMMAND: COMMAND_M_MULTI_PRESS_COMPLETE,
+        ARGS: {
+            ATTRIBUTE_ID: 84,
+            ATTRIBUTE_NAME: ATTRIBUTE_ACTION,
+            BUTTON: TURN_OFF,
+            VALUE: 20,
+        },
     },
-    (LONG_PRESS, TURN_ON): {
+    (LONG_RELEASE, TURN_ON): {
```

Comment:
This would break existing automations with the "long press" trigger, right?

Reply:
Yes

Reply:
Indeed. The automation using them will show an error. Easy fix tho.

Is there a deprecation mechanism in place in ha? Or worse case I guess we can alias? It's really when we release the button that it sends either a long or short press event :/

Reply:
There is not, unfortunately. What do you think about changing this back for now? Only real difference between them is another UI text for the end-user.

Reply:
If it's the only thing that breaks backward compatibility, I'm fine. If someone was  directly listening too zha_event, there are more breaking changes tho... Since the command changed too. Not sure if it's acceptable?

Reply:
I think my last commit fixes this concern. I'll leave you the final word.